ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/configuration/CommandClientConfiguration.scala

```diff
@@ -12,13 +12,21 @@ import java.time.Duration
   * @param defaultDeduplicationTime The deduplication time to use for commands that do not have
   *                                 a deduplication time set. The deduplication time is also used
   *                                 as the time after which commands time out in the command client.
+  * @param submissionIdPropagationSupported Tells whether submission IDs are being returned in Completions.
+  *                                         Should be false for integrations where the mutating schema is enabled.
   */
 final case class CommandClientConfiguration(
     maxCommandsInFlight: Int,
     maxParallelSubmissions: Int,
     defaultDeduplicationTime: Duration,
+    submissionIdPropagationSupported: Boolean,
```

Comment:
I don't have access to the information about the schema on the client-side. I'm open to other ideas.

Reply:
A possibly crazy question: Can we just change the mutating schema to use submission IDs easily? @digital-asset/kv-participant?

Reply:
As pointed out by @rautenrieth-da [here](https://github.com/digital-asset/daml/pull/11011#r719727805), the mutating schema is already deprecated so we can just disable the tests that depend on submission ID propagation.